magento/magento2
# magento/magento2#22612: Export of Downloadable Products type

- Export of Downloadable Product Links
- Export of Sample Links




Export of Downloadable Products type
### Description (*)

Export of Downloadable type product, which will help customers to export data of products with type downloadable.
### Fixed Issues (if relevant)

1. magento/magento2#22612: Export of Downloadable Products type.

### Manual testing scenarios (*)

1. Login to Magento2 Admin
2. Create a downloadable type of product from the catalog product menu.
3. Go to System > Export > Select Products.
4. Click on Continue.
5. The exported file should have the information about Downloadable product under the column headers **downloadable_links** and **downloadable_samples**
[Export File](https://www.dropbox.com/s/peplrasigdb5ogs/downloadableProducts.csv?dl=0)

### Contribution checklist (*)
